crypto/fipsmodule/rand/rand.c

```diff
@@ -227,9 +227,20 @@ static void CRYPTO_get_fips_seed(uint8_t *out_entropy, size_t out_entropy_len,
     abort();
   }
 
-  // Generate the required number of bytes with Jitter.
-  if (jent_read_entropy_safe(&state->jitter_ec, (char *) out_entropy,
-                             out_entropy_len) != (ssize_t) out_entropy_len) {
+  // |jent_read_entropy| has a false positive health test failure rate of 2^-22.
+  // To avoid aborting so frequently, we retry 3 times.
+  size_t num_tries;
+  const size_t max_num_tries = 3;
+  for (num_tries = 0; num_tries < max_num_tries; num_tries++) {
```

Comment:
Can you make `CALL_RDRAND_WITH_RETRY` more general and use that?

Reply:
I could have, but I was informed that we need to reset the Jitter object upon failure (see the latest update). With that, I don't see how `CALL_RDRAND_WITH_RETRY` can be used cleanly.